File: Exemplos Linguagens/Soma.java
import java.util.Scanner;

public class SomaNumeros {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        // Lê dois números inteiros
        System.out.print("Digite o primeiro número: ");
        int num1 = scanner.nextInt();
        System.out.print("Digite o segundo número: ");
        int num2 = scanner.nextInt();

        // Calcula a soma
        int sum = num1 + num2;

        // Exibe o resultado
        System.out.println("A soma é: " + sum);
    }
}

File: Exemplos Linguagens/Soma.c
#include <stdio.h>

int main() {
    int num1, num2, sum;

    // Lê dois números inteiros
    printf("Digite o primeiro número: ");
    scanf("%d", &num1);
    printf("Digite o segundo número: ");
    scanf("%d", &num2);

    // Calcula a soma
    sum = num1 + num2;

    // Exibe o resultado
    printf("A soma é: %d\n", sum);

    return 0;
}
